modules/group_alert_banner/group_alert_banner.group.permissions.yml

```diff
@@ -0,0 +1,3 @@
+access localgov_alert_banner overview:
+  title: 'Access Alert banner listing page'
+  description: 'Access the Alert banner listing and management page, regardless of banner type.'
```

Comment:
This sounds like the kind of general permission we should have in the main module, how are we access controlling it at the moment? Why do we need the seperate permission?

Reply:
So I think I'm understanding that this is for the seperate group view, which is different from the main admin view. And we only want to grand group insiders the group admin view and not the main one?
(This is me trying to understand localgov microsites permissions)

Reply:
> And we only want to grant group insiders the group admin view and not the main one?

That's right.